src/material-examples/form-field-custom-control/example-tel-input-example.html

```diff
@@ -1,7 +1,7 @@
 <div [formGroup]="parts" class="example-tel-input-container">
-  <input class="example-tel-input-element" formControlName="area" size="3">
+  <input class="example-tel-input-element" formControlName="area" size="3" aria-label="Enter first 3 digits of phone number">
```

Comment:
The label should be "Area code".

Reply:
Changed as per your suggestion. Can you please check now.